### PR TITLE
Feature/unit tests

### PR DESCRIPTION
**Pull request by lepsalex**
Unit Tests: ApplicationService #107 
Unit Tests: GroupService #108 
Unit Tests: UserService #109

### CODE REVIEW

File: .env.schema [hunk 1]
```diff
@@ -1,9 +1,13 @@
+EGO_ACTIVE_PROFILES=
+EGO_SERVER_PORT=
 EGO_DB_HOST=
 EGO_DB_PORT=
 EGO_DB=
 EGO_DB_USER=
 EGO_DB_PASS=
 EGO_SERVER_GOOGLE_CLIENT_IDS=
+EGO_SERVER_FACEBOOK_APP_ID=
+EGO_SERVER_FACEBOOK_SECRET=
```
Comment (andricDu) [inline .env.schema]: :+1:

File: src/main/java/org/overture/ego/controller/resolver/PageableResolver.java [hunk 2]
```diff
@@ -42,6 +42,15 @@ public Object resolveArgument(MethodParameter methodParameter,
     String sort = nativeWebRequest.getParameter("sort");
     String sortOrder = nativeWebRequest.getParameter("sortOrder");
 
+    return getPageable(limit, offset, sort, sortOrder);
+
```
Comment (andricDu) [inline src/main/java/org/overture/ego/controller/resolver/PageableResolver.java]: delete this newline

**Comment by andricDu** [comment]
@lepsalex once you resolve the conflicts I'll merge.

**Comment by lepsalex** [comment]
All good now

**Pull request merged**